content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -39,31 +39,37 @@ model where `Pods` can be treated much like VMs or physical hosts from the
 perspectives of port allocation, naming, service discovery, load balancing,
 application configuration, and migration.
 
+Kubernetes IP addresses exist at the `Pod` scope - containers within a `Pod`
+share their network namespaces - including their IP address and MAC address.
+This means that containers within a `Pod` can all reach each other's ports on
+`localhost`. This also means that containers within a `Pod` must coordinate port
+usage, but this is no different from processes in a VM.  This is called the
+"IP-per-pod" model.
+
+In every cluster, there exists an abstract pod-network to which pods
+are connected by default, unless explicitly configured to use the
+host-network (which is an optional capability). Even if a host have
```

Comment:
not clear what "(which is an optional capability)" means. I'm guessing you're talking about the fact that the network plugin might not support it, but given that we're mostly talking about pods here it sounds more like you mean "it's optional for pods to specify that they want host-network", which you already just said and so it sounds weird.

maybe "(on platforms that support it)"?

Reply:
Sure - the point was that the ability to specify host-network is optional, but i like your words beter

Reply:
done